packages/auth/test/helpers/integration/helpers.ts

```diff
@@ -93,3 +95,24 @@ function stubConsoleToSilenceEmulatorWarnings(): sinon.SinonStub {
     }
   });
 }
+
+export function getTotpCode(
+  sharedSecretKey: string,
+  periodSec: number,
+  verificationCodeLength: number
+): string {
+  const token = totp(sharedSecretKey, {
+    period: periodSec,
+    digits: verificationCodeLength,
+    algorithm: 'SHA-1'
+  });
+
+  return token;
+}
+
+export function delay(dt: number): Promise<void> {
+  return new Promise(resolve => setTimeout(resolve, dt));
+}
+
+export const email = 'totpuser-donotdelete@test.com';
+export const incorrectTotpCode = '1000000';
```

Comment:
nit: can we make the incorrect code obvious like "abcd"? Or add a comment like "1000000" is always incorrect since it has 7 digits and we expect 6.